app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js

```diff
@@ -33,7 +33,7 @@ const MigrationsCompletedList = ({
             const failed = mostRecentRequest && mostRecentRequest.status === 'Error';
 
             const tasks = {};
-            mostRecentRequest.miq_request_tasks.forEach(task => {
+            if(mostRecentRequest) mostRecentRequest.miq_request_tasks.forEach(task => {
```

Comment:
how about -
```js
mostRecentRequest && mostRecentRequest.miq_request_tasks.forEach(task => {
```

like it's done above

Reply:
intentionally and for a very good reason
<img width="687" alt="screen shot 2018-06-28 at 8 37 13 pm" src="https://user-images.githubusercontent.com/6640236/42067219-19a15c20-7b13-11e8-9698-c88aaf624cdf.png">


eslint is not a fan

Reply:
Ok, fair.